unpackaged/config/rd2_post_config/objects/npe03__Recurring_Donation__c.object

```diff
@@ -322,6 +322,43 @@
         <trackTrending>false</trackTrending>
         <type>Date</type>
     </fields>
+    <fields>
+        <fullName>%%%NAMESPACE%%%Status__c</fullName>
+        <description>Indicates if this Recurring Donation is actively in use, temporarily suspended, pause, or closed.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>Indicates if this Recurring Donation is actively in use, temporarily suspended, pause, or closed.</inlineHelpText>
+        <label>Status</label>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+        <valueSet>
+            <restricted>true</restricted>
+            <valueSetDefinition>
+                <sorted>false</sorted>
+                <value>
+                    <fullName>Active</fullName>
+                    <default>true</default>
+                    <label>Active</label>
+                </value>
+                <value>
+                    <fullName>Lapsed</fullName>
+                    <default>false</default>
+                    <label>Lapsed</label>
+                </value>
+                <value>
+                    <fullName>Closed</fullName>
+                    <default>false</default>
+                    <label>Closed</label>
+                </value>
+                <value>
+                    <fullName>Pause</fullName>
+                    <default>false</default>
+                    <label>Pause</label>
+                </value>
```

Comment:
@force2b Does it make sense to add the new Status value in the post config npe__Recurring_Donation__c.Object or it should exist in the default .Object file. I add it to here because I think it's similar to Last Day Of Month value on Day_Of_Month__c field. However, when we added the Lapsed, we put it into the default npe__Recurring_Donation__c.Object